db/seeds.rb

```diff
@@ -1,8 +1,10 @@
 
 puts "Creating playlists..."
 
-Playlist.create(name: "Playlist1", energy: "60", danceability: "65", genre: "rock")
-Playlist.create(name: "Playlist2", energy: "25", danceability: "30", genre: "pop")
-Playlist.create(name: "Playlist3", energy: "45", danceability: "40", genre: "acoustic")
+User.create(email: "test@test.com", password: "coucou")
+
+Playlist.create(name: "Playlist1", energy: "60", danceability: "65", genre: "rock", user_id: 1)
+Playlist.create(name: "Playlist2", energy: "25", danceability: "30", user_id: 1)
+Playlist.create(name: "Playlist3", energy: "45", danceability: "40", user_id: 1)
```

Comment:
```suggestion
user = User.create(email: "test@test.com", password: "coucou")

user.playlists.create(name: "Playlist1", energy: "60", danceability: "65", genre: "rock")
user.playlists.create(name: "Playlist2", energy: "25", danceability: "30")
user.playlists.create(name: "Playlist3", energy: "45", danceability: "40")
```

ton `user_id: 1` est bancal: si tu delete et re seed ça ne marchera pas